test-charts/web-service/reloader/values.yaml

```diff
@@ -0,0 +1,11 @@
+bandstand-web-service:
+  owner: ktech-developer-platform
+  env: test
+  business: ktech
+
+  ingress:
+    enabled: true
+
+  reloader:
+    enabled: false
```

Comment:
```suggestion
    enabled: true
```

Reply:
not sure why I just change this back to false again